script/Arcueid/Day 10/10_00_ARC10_7_2B.txt

```diff
@@ -75,7 +75,7 @@ I don't spot any people smeared head to toe in them, as I had before.
 [sha:d41b8b4de8ae6598240ce1213df1f5f296c2ff14]{
 -- Page 778, Offset 26400.
 -- こめかみを指で押さえる。
-I massage my temple with my fingers.
+I press on my temples with my fingers.
```

Comment:
Massage is much more common an expression for this in EN I feel. Plural fine though.